models/entities/person.go

```diff
@@ -80,7 +84,45 @@ func NewPerson(store *Store, id string) (*Person, error) {
 		return nil, err
 	}
 
-	// Revisit: Create Inbox/Outbox/etc
+	// TOOD: These hould be Inbox/Outbox kinds but this will do For Now (TM)
+	inbox, err := NewStream(store, id+"/inbox")
+	if err != nil {
+		return nil, errors.Wrap(err, "Creating inbox")
+	}
+
+	outbox, err := NewStream(store, id+"/outbox")
+	if err != nil {
+		return nil, errors.Wrap(err, "Creating outbox")
+	}
+
+	inbox.AttributedTo = jsonld.ToRef(id)
+	outbox.AttributedTo = jsonld.ToRef(id)
+	obj.Inbox = jsonld.ToRef(inbox.GetID())
+	obj.Outbox = jsonld.ToRef(outbox.GetID())
+
+	creation, err := NewActivity(store, outbox, []string{as2("Create")})
+	if err != nil {
+		return nil, errors.Wrap(err, "Creating Create activity")
+	}
+
+	creation.Actor = jsonld.ToRef(id)
+	creation.Obj = jsonld.ToRef(id)
+
+	if err := store.Save(inbox); err != nil {
```

Comment:
Thought: we could use `multierr` for this:

```
return obj, multierr.Combine(
  errors.Wrap(store.Save(inbox), "Saving inbox"),
  errors.Wrap(store.Save(outbox), "Saving outbox"),
  // ...
)
```

Reply:
It feels silly to have this silly long list of `if err := ...; err != nil { return err }`! It's like half the function body >_<